ffi-build.sh

```diff
@@ -54,9 +62,19 @@ cp -v LICENSE LICENSE-3rdparty.yml NOTICE "$destdir/"
 
 export RUSTFLAGS="${RUSTFLAGS:- -C relocation-model=pic}"
 
-echo "Building the libddprof_ffi.a library (may take some time)..."
+echo "Building the libddprof_ffi library (may take some time)..."
 cargo build --release --target "${target}"
-cp -v "target/${target}/release/libddprof_ffi.a" "$destdir/lib/"
+cp -v "target/${target}/release/libddprof_ffi.a" "target/${target}/release/libddprof_ffi${shared_library_suffix}" "$destdir/lib/"
+
+if [[ "$remove_rpath" -eq 1 ]]; then
+    patchelf --remove-rpath "$destdir/lib/libddprof_ffi${shared_library_suffix}"
+fi
+
+# objcopy might not be available on OSX
```

Comment:
The most minor of nitpicks: It's "macOS" nowadays, not "OSX" (it's not even 10 anymore :P)

Reply:
Fixed ;)